continuous_integration/install.sh

```diff
@@ -47,7 +47,15 @@ if [[ "$DISTRIB" == "conda" ]]; then
 
 elif [[ "$DISTRIB" == "ubuntu" ]]; then
     # Use standard ubuntu packages in their default version
-    sudo apt-get install -qq python-scipy python-nose python-pip
+    sudo apt-get install -qq python-scipy
+    # At the time of writing numpy 1.9.1 is included in the travis
+    # virtualenv but we want to used numpy installed through apt-get
+    # install.
+    deactivate
+    # Create a new virtualenv using system site packages for numpy and scipy
+    virtualenv --system-site-packages testvenv
+    source testvenv/bin/activate
+    pip install nose
```

Comment:
> In this case we could have just disabled the venv provided by travis (with deactivate) and use the system nosetests command from /usr/bin/nosetests

Something I forgot to mention that made me go for a new virtualenv even if it wasn't entirely necessary in the scikit-learn case since (as mentioned earlier) all the dependencies are installed through apt-get install: the line installing coverage and coversall doesn't contains sudo, so it will fail if you are working with your system python. At the time I just thought it is more uniform if all the DISTRIB end up creating a fresh environment so that COVERAGE=true works for all of them.